website/docs/docs/cloud/configure-cloud-cli.md

```diff
@@ -95,6 +95,7 @@ To set environment variables in the dbt Cloud CLI for your dbt project:
 ## Use the dbt Cloud CLI
 
 - The dbt Cloud CLI uses the same set of [dbt commands](/reference/dbt-commands) and [MetricFlow commands](/docs/build/metricflow-commands) as dbt Core to execute the commands you provide. For example, use the [`dbt environment`](/reference/commands/dbt-environment) command to view your dbt Cloud configuration details.
+- You can run multiple different invocations or commands in parallel. For example, `dbt build` and `dbt parse`.  Note, that you're unable to run the same dbt commands in parallel. For example, running `dbt build` at the same time isn't supported.
```

Comment:
It's not 100% accurate, on backend we categorize dbt commands into two types, data warehouse write and data warehouse non-write. 

Data warehouse write command always has 1 parallelism. E.g. build.
Data warehouse non-write command can have x(today it's 1, but it might be more, let's say 2) parallelism. E.g. parse.

I'm not sure how to phrase this as it's too complicated for customer to understand, maybe say you are not able to run data warehouse conflicted commands?